<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EasyPic</class>
 <widget class="QWidget" name="EasyPic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyPic</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="localFileBtn">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>本地文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="preBtn">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>上一张</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextBtn">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>下一张</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addBtn">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteBtn">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="showLabel">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
